pkg/vpc/vpc_test.go

```diff
@@ -91,4 +102,82 @@ var _ = Describe("VPC Endpoints", func() {
 			Error:       errors.New(eks.ErrCodeResourceNotFoundException),
 		}),
 	)
+
+	DescribeTable("can import VPC",
```

Comment:
Can we add a test that checks that given an input that would normally produce duplicate subnets checks that there are no dups?

Reply:
Agreed that this test case must be added. However, all the methods in this file is public package methods, so it's hard to mock the entire method stack. 

So just want to check with you if I can refactor the code a little bit with dependency injection for private method like this commit https://github.com/weaveworks/eksctl/pull/1477/commits/1905d0b667c118b352943e629d0b253008b78cd3